roles/openshift_node_dnsmasq/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dns_domain: 'cluster.local'
+kube_svc_ip: '172.30.0.1'
```

Comment:
We probably don't want to use defaults here, otherwise we might end up with a case where a refactoring would end up with these values not being overridden properly.